wpilibj/src/main/java/edu/wpi/first/wpilibj/Watchdog.java

```diff
@@ -26,14 +26,14 @@
 @SuppressWarnings("PMD.TooManyMethods")
 public class Watchdog implements Closeable, Comparable<Watchdog> {
   // Used for timeout print rate-limiting
-  private static final long kMinPrintPeriod = 1000000; // us
+  private static final long kMinPrintPeriodMicroS = 1000000; // us
```

Comment:
I personally don't think it's worthwhile to rename these members. These aren't user-facing, and the comment should be enough of a reminder.

Reply:
I might agree, except that in doing that, I found at least one place where the units in the comment didn't match the code.